editor/css/dark.css

```diff
@@ -154,7 +189,7 @@ select {
 #sidebar {
 	position: absolute;
 	right: 0;
-	top: 32px;
```

Comment:
`32px` was fine. The problem is the new border that has been added to Panel, which is affecting `menubar`.